### PR TITLE
Display tags in commit list

### LINKED ISSUE #9427: Display tags in commit list and commit details
From @ampinsk:

We decided to prototype with this design and may adjust depending on how this looks with real data

View-Tags-07@2x

_____
part of work for #9424

### PR DESCRIPTION
**Pull request by outofambit**
Closes #9427
(redo of #9488)

### Screenshots

Screen Shot 2020-04-20 at 8 26 29 AM
Screen Shot 2020-04-20 at 8 25 38 AM
Screen Shot 2020-04-20 at 8 26 04 AM
Screen Shot 2020-04-21 at 1 49 51 PM

### CODE REVIEW

File: app/styles/ui/_avatar-stack.scss [hunk 1]
```diff
@@ -64,7 +64,7 @@
     box-sizing: content-box;
     margin-right: -11px;
     background-color: inherit;
-    box-shadow: 1px 0 0 $white;
+    box-shadow: 1px 0 0 var(--background-color);
```
Comment (outofambit) [inline app/styles/ui/_avatar-stack.scss:67]: this is a bonus fix from @ampinsk for our avatar stacks! ✨

**Comment by outofambit** [comment]
@tierninho thank you! i've fixed that and put a screenshot of that state in the pr description now.

**Comment by rafeca** [comment]
I found a small issue when a commit has the unpushed indicator:

image

gonna try to tweak the styles a bit to fix it

**Comment by outofambit** [comment]
thanks @rafeca  this looks good to me :)

**Pull request merged**